tests/nfs-bug-5140/test.yaml

```diff
@@ -0,0 +1,12 @@
+pcap: nfsv2.pcap
+
```

Comment:
We should ensure that we are only running this test for Suricata version 9, since that's the version that will have the patch, for now.